privacy-snapshot.json

```diff
@@ -52,6 +52,7 @@
   "static.cx.metamask.io",
   "swap.api.cx.metamask.io",
   "test.metamask-phishing.io",
+  "thumb.canalplus.pro",
   "token.api.cx.metamask.io",
```

Comment:
I'm still not sure where "thumb.canalplus.pro" is coming from;

Reply:
Removed [14a7051](https://github.com/MetaMask/metamask-extension/pull/27629/commits/14a70515c6dfc0ecc3ff589ff347c51644df2048)